backend/src/libs/packages/encrypt/encrypt.service.ts

```diff
@@ -0,0 +1,25 @@
+import bcrypt from 'bcryptjs';
+
+class EncryptService {
```

Comment:
also it should implement interface as other packages
```suggestion
class BaseEncrypt implements Encrypt {
```

Reply:
In approved JWT service there is no "base" implementation. Is it ok?